lib/everypolitician_extensions.rb

```diff
@@ -52,6 +58,32 @@ def top_identifiers
                            .sort_by { |s, ids| [-ids.size, s] }
                            .map { |s, _ids| s }
     end
+
+    private
+
+    def current_cabinet_memberships
```

Comment:
I'm not entirely sure what this method is meant to be for, and the implementation is a little hard to follow, but I _think_ the "current" is a nismomer.